enterprise/constants.py

```diff
@@ -28,3 +28,24 @@
 
 # Course mode sorting based on slug
 COURSE_MODE_SORT_ORDER = ['verified', 'professional', 'no-id-professional', 'audit', 'honor']
+
+PROGRAM_TYPE_DESCRIPTION = {
```

Comment:
Should these details be configurable somehow? For example, to be able to add blurbs for other program types.

Reply:
They were originally supposed to be through a new `description` field on discovery, but that got scratched. CC @douglashall for more details.